src/ImageService.Broker/Consumers/CreateImagesConsumer.cs

```diff
@@ -41,6 +42,9 @@ private async Task<object> CreateImagesAsync(ICreateImagesRequest request)
           return null;
         }
 
+        (bool isSuccess, string compressedContent, string extension) imageCompressResult =
+          await _compressHelper.CompressAsync(imageResizeResult.resizedContent, imageResizeResult.extension, 10);

```

Comment:
10Kb for avatars (ImageSource.User); for other images weight can be another
for now add compressing only for avatars